problems/removeVowels.js

```diff
@@ -6,8 +6,8 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+function removeVowels(str) {
+    return str.replace(/[aeiou]/gi, "");
```

Comment:
Nice RegEx but make sure you truly understand why it's working. This test in feeling should be all about the why it's working, not just get something that works. Make sure you can replicate this answer during a coding interview.